### ISSUE TITLE
Missing title-tag in meta informations

### ISSUE THREAD

**Issue by Balindil** [contao/core-bundle#601]
In meta informations of images title-tag is replaced by alternate text. So I can't give images a title tag and there is no caption in the lightbox presentation.

**Comment by leofeyer** [comment]
What exactly did you do and what exactly do you want to accomplish?

**Comment by Balindil** [comment]
Up to Contao 3.5.x in file manager you can write the title tag in meta informations of images. This field is disappeared. Instead there is now the field "Alternate text" which is not the same.

**Comment by leofeyer** [comment]
AFAIR, the field was always used as `alt` attribute, even though it is called `title` in Contao 3. We have just fixed the wrong label in Contao 4.

**Comment by Balindil** [comment]
I don't think so. I always used it for the `title` tag. Where else could I enter the title for images in a gallery? The `title` tag is also used as image caption in the lightbox.

**Comment by leofeyer** [comment]
<img width="744" alt="" src="https://cloud.githubusercontent.com/assets/1192057/19646109/51b00056-99f8-11e6-98a8-6316ab0a5309.png">

<img width="684" alt="" src="https://cloud.githubusercontent.com/assets/1192057/19646116/590aedf2-99f8-11e6-8ee3-9c3755db92a4.png">

**Comment by leofeyer** [comment]
It is true that in Contao 3, we are also using the alternative text as link title, which however is wrong, because the link title should describe the link and the alternative text the image.

Anyways, you can easily change this behavior in the templates. You can even add additional meta fields in the DCA if necessary.

**Comment by fritzmg** [comment]
@Balindil you could change the default `j_colorbox` template to
```php
<?php

// Add the colorbox style sheet
$GLOBALS['TL_CSS'][] = 'assets/colorbox/css/colorbox.min.css|static';

?>

<script src="<?= TL_ASSETS_URL ?>assets/colorbox/js/colorbox.min.js"></script>
<script>
  jQuery(function($) {
    $('a[data-lightbox]').map(function() {
      $(this).colorbox({
        // Put custom options here
        loop: false,
        rel: $(this).attr('data-lightbox'),
        maxWidth: '95%',
        maxHeight: '95%',
        title: $(this).find('img').attr('alt')
      });
    });
  });
</script>
```
This (somewhat) restores the behaviour of Contao 3.

**Comment by asaage** [comment]
you have ```title="Voting"``` in the cboxElement (hidden by the green Arrow) which should make it the title of the lightbox.

**Comment by fritzmg** [comment]
Currently, Contao 4 will not show any kind of title in the colorbox of a gallery.

**Comment by asaage** [comment]
add ```title``` [here](https://github.com/contao/core-bundle/blob/master/src/Resources/contao/templates/gallery/gallery_default.html5#L9)

**Comment by leofeyer** [comment]
For the record, we have changed this in #4413. And we should probably provide two fields, one for the title and one for the alternate text.

**Comment by asaage** [comment]
not very happy with [#4413](https://github.com/contao/core-bundle/commit/d3fb804db04922955212dbb3d3bdf75c811182a1)...
what happens if people update from 3.5?
Everybody who painfully edited titles for their lightbox-enabled galleries will find that as alternate text in 4.2.4 -> edit the gallery- or colorbox template to fix their lightboxes and fill out additional field ```title``` in 4.3 to fix it again.

**Comment by leofeyer** [comment]
@asaage I totally agree. What would you suggest?

@contao/developers /cc

**Comment by asaage** [comment]
honestly i'm ok with the way it was before the change

```html
<figure>
  <a data-lightbox title="MSC.aw_title" href="foo">
    <img alt="MSC.aw_title" src="bar.jpg">
  </a>
<figure>
```
occasionally i even rendered ```caption``` as img alt attribute and deleted the ```figcaption``` part.

Might be better to 
- revert aw_title back to Title 
- add ```alternate text``` later (if needed) which falls back to title (or caption) if not provided.

**Comment by leofeyer** [comment]
Implemented in dfea0900308bc2261651df7cbb9710871ec70d14.
